src/process/process.rs

```diff
@@ -63,8 +63,9 @@ pub struct Callback {
     pub pc: usize
 }
 
-#[repr(C,packed)]
```

Comment:
Why not `packed`? Since they are all `usize`s, it doesn't matter in practice, but aren't there no guarantees of the spacing between them otherwise? My concern is that we're generating the binary on a totally different platform than where we're reading it, so we have to be extra careful we know exactly how it gets laid out by the compiler.